crepecake/src/test/resources/json/basic.json

```diff
@@ -0,0 +1 @@
+{"number":54,"text":"crepecake","digest":"sha256:8c662931926fa990b41da3c9f42663a537ccd498130030f9149173a0493832ad","innerObject":{"number":23,"texts":["first text","second text"],"digests":["sha256:91e0cae00b86c289b33fee303a807ae72dd9f0315c16b74e6ab0cdbe9d996c10","sha256:4945ba5011739b0b98c4a41afe224e417f47c7c99b2ce76830999c9a0861b236"]},"list":[{"number":42,"texts":[]},{"number":99,"texts":["some text"],"digests":["sha256:d38f571aa1c11e3d516e0ef7e513e7308ccbeb869770cb8c4319d63b10a0075e"]}]}
```

Comment:
would be nice to format this more legibly with spaces and indents and line breaks

Reply:
The main problem with formatting this is that it won't be able to match the output of `JsonHelper#writeJson` as that will stream the JSON in the minified form. The solution would be

- to add an option to the `writeJson` method to be able to pretty print or 
- have the test minify the basic.json after reading it
- leave it as is, since the file can be read legibly with any pretty JSON reader

LYMWYT

Reply:
Ideally we wouldn't fail the test on differences in insignificant white space -- i.e. use a smarter comparison function -- but for now this will work.